src/Simplex/Chat/ProfileGenerator.hs

```diff
@@ -0,0 +1,3163 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Simplex.Chat.ProfileGenerator where
+
+import Data.Text (Text)
+import Simplex.Chat.Types (Profile (..))
+import System.Random (randomRIO)
+
+generateRandomProfile :: IO Profile
+generateRandomProfile = do
+  adjective <- pick adjectives
+  noun <- pickNoun adjective 2
+  pure $ Profile {displayName = adjective <> noun, fullName = "", image = Nothing}
+  where
+    pick :: [a] -> IO a
+    pick xs = (xs !!) <$> randomRIO (0, length xs - 1)
+    pickNoun :: Text -> Int -> IO Text
+    pickNoun adjective n
+      | n == 0 = pick nouns
+      | otherwise = do
+        noun <- pick nouns
+        if noun == adjective
+          then pickNoun adjective (n - 1)
+          else pure noun
+
+adjectives :: [Text]
```

Comment:
probably better to turn off ormolu and make a smaller number of long lines, like 20 names per line